business_objects/attribute.py

```diff
@@ -92,7 +92,7 @@ def get_text_attributes(
     )
     if state_filter:
         query = query.filter(Attribute.state.in_(state_filter))
-    text_attributes = query.all()
+    text_attributes = query.order_by(Attribute.relative_position.asc()).all()
```

Comment:
not sure why we order and then put it in a dictionary (most programming languages define dictionaries as unordered, python since 3.7 has it ordered however so it will impact an iteration over the dict to some extent).

If it's for iteration purposes then we can just keep it, just wanted to be sure that it's intended

Reply:
yes it is on purpose